guide/en/structure/package.md

```diff
@@ -65,43 +65,60 @@ The following example shows the `composer.json` file for the `yiisoft/yii-widget
         "forum": "https://www.yiiframework.com/forum/",
         "wiki": "https://www.yiiframework.com/wiki/",
         "irc": "irc://irc.freenode.net/yii",
+        "chat": "https://t.me/yii3en",
         "source": "https://github.com/yiisoft/yii-widgets"
     },
-    // package dependencies
-    "require": { 
-        "php": "^7.4",
-        "yiisoft/cache": "^3.0@dev",
-        "yiisoft/view": "^3.0@dev",
-        "yiisoft/widget": "^3.0@dev"
+    "funding": [
+        {
+            "type": "opencollective",
+            "url": "https://opencollective.com/yiisoft"
+        },
+        {
+            "type": "github",
+            "url": "https://github.com/sponsors/yiisoft"
+        }
+    ],
+    "require": {
+        "php": "^7.4|^8.0",
```

Comment:
Should be `^8.0`?

Reply:
This is real example of `composer.json` from `yiisoft/yii-widget`: https://github.com/yiisoft/widget/blob/master/composer.json